cmd/fixtures_test/struct_utils.go

```diff
@@ -57,7 +66,14 @@ func UpdateSenderKeyToAddress(bytes []byte, t *testing.T) []byte {
 func UpdateReceiverKeyToAddress(bytes []byte, t *testing.T) []byte {
 	raw := UnmarshalIntoEmptyInterface(bytes, t)
 
-	receiverName, ok := raw["Receiver"].(string)
+	receiverTempName, ok := raw["Receiver"].(string)
+	receiverNameIndex, err := strconv.Atoi(strings.TrimLeft(receiverTempName, "account"))
+	t.MustNil(err, "temp account name doesn't match to the account args")
+	receiverNameIndex--
+
+	t.MustTrue(receiverNameIndex < len(accountNames), "temp account name doesn't match to the account args")
+	receiverName := accountNames[receiverNameIndex]
```

Comment:
account replacement section code are all same. There should be a utility function and call it.